nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -27,6 +27,14 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - `services.sourcehut.dispatch` and the corresponding package (`sourcehut.dispatchsrht`) have been removed due to [upstream deprecation](https://sourcehut.org/blog/2022-08-01-dispatch-deprecation-plans/).
 
+- The `snapserver` module option `openFirewall` default value has been changed from `true` to `false`. You will need to explicitely set this option to `true`, or configure your firewall.
```

Comment:
Nit: I'm a fan of linking options whenever referencing them. I.e. to write `[services.snapserver.openFirewall](#opt-services.snapserver.openFirewall)`. With that it's possible for the reader to directly navigate to the option's manual.

Reply:
I'll do that, and this even caught an issue! The module for avahi is `avahi` and not `avahi-daemon`.